spec/fixtures/build.gradle.kts

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
This build.gradle.kts file was generated via `gradle init` and selecting a Kotlin application with the Kotlin gradle DSL.